src/main/java/org/folio/holdingsiq/service/LoadService.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.holdingsiq.service;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.holdingsiq.model.Holdings;
+import org.folio.holdingsiq.model.HoldingsLoadStatus;
+
+public interface LoadService {
+
+  /**
+   * Populates holdings data to a staging area
+   */
+  CompletableFuture<Void> populateHoldings();
```

Comment:
Could you please let me know more details about the purpose of this method

Reply:
according to the doc - https://developer.ebsco.com/interactive/holdingsiq there three steps needed to be performed :
1 - POST holdings to the stage area
2 - GET status of the snapshot
3 - GET holdings if status from the previous step is "Completed"

Reply:
thanks, got it